go/.go_package_deps_darwin

```diff
@@ -603,6 +603,7 @@
       "github.com/keybase/client/go/kbfs/test" : 1,
       "github.com/keybase/client/go/kbfs/tlf" : 1,
       "github.com/keybase/client/go/kbhttp" : 1,
+      "github.com/keybase/client/go/kbnm" : 1,
```

Comment:
what's going on here?  what is this .go_package_deps_darwin file?

Reply:
Yeah that's what I did, which resulted in these change. I think Patrick was asking what this is for.

Reply:
Oh right you're asking what's going on. These files now contain the dependency linkages between packages. This allows CI to perform tests such that only the minimal set of tests is executed in CI, generally improving performance.

Reply:
We could exclude them from the code, but then we'd have to add a hard 2 minute step into CI that can't be executed in parallel with other steps. So instead we run that step in parallel and notify the engineer if the dependencies need to be recalculated.

Reply:
I will add this to the README.